mongodbatlas/data_source_mongodbatlas_federated_settings_connected_organization_test.go

```diff
@@ -0,0 +1,71 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceMongoDBAtlasFederatedSettingsOrganizationConfig_basic(t *testing.T) {
+	SkipTestExtCred(t)
+	var (
+		resourceName        = "data.mongodbatlas_cloud_federated_settings_org_config.test"
+		federatedSettingsID = os.Getenv("MONGODB_ATLAS_FEDERATION_SETTINGS_ID")
+		orgID               = os.Getenv("MONGODB_ATLAS_FEDERATED_ORG_ID")
```

Comment:
Can you add these to the repo's README please?  You'll see lots of examples in there to follow.

Reply:
Added